pandas/core/frame.py

```diff
@@ -6814,7 +6814,6 @@ def sort_values(
     ) -> None:
         ...
 
-    # TODO: Just move the sort_values doc here.
     # error: Signature of "sort_values" incompatible with supertype "NDFrame"
     @Substitution(**_shared_doc_kwargs)
```

Comment:
Also whatever values are being substituted with this call should just be in-lined in the docstring

Reply:
Hello,
I want to apologize for replying only now. I first encountered problems with creating a development environment for building the documentation locally and then got struck down with Covid.

I'm now back on track and will take care of this.

Cheers

Reply:
No worries (and no need to apologize). Glad you are feeling better!